data_collection/gazette/spiders/mg_belo_horizonte.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+import re
+
+import scrapy
+from dateutil.rrule import DAILY, rrule
+from urllib.parse import urlencode
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class MgBeloHorizonteSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3106200"
+
+    name = "mg_belo_horizonte"
+    allowed_domains = ["dom-web.pbh.gov.br"]
+    start_date = datetime.datetime(2000, 1, 2)
```

Comment:
> 1995-09-26

Thanks! I tried to find the oldest date, but you did a better job than me :-)

Reply:
> Also, `start_date` as `datetime.datetime` is giving errors.
> 
> ```
>   File ".../querido-diario/raspadores/data_collection/gazette/pipelines.py", line 17, in process_item
>     if spider.start_date > item.get("date"):
> TypeError: can't compare datetime.datetime to datetime.date
> ```
> 
> This can be a `datetime.date` already, eliminating the need for a `rule_start_date` variable

`rrule` returns a `datetime`, so we need to convert it to `date` so it can be compared with `self.start_date` (as `date`). This was already fixed. I didn't understando why you get these errors, as they don't happen to me.

Reply:
It wasn't happening when comparing `rrule` dates with `self.start_date` but in the pipeline when the `date` attribute in the `Gazette` item was compared against `self.start_date`. But now it's fixed, thanks :)